test/test-greet.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const expect = require('chai').expect;
+
+let greetings = require('../lib/greet.js');
+
+describe('greetings', function(){
+  it('should exist', function(){
+    expect(greetings).to.not.be.undefined;
```

Comment:
Because JavaScript is javascripty, you can test for truthiness. does this value exist? expect(greetings).to.be...what?  Just another little testing tip :)